liota/dcc_comms/websocket_dcc_comms.py

```diff
@@ -41,13 +41,15 @@
 
 class WebSocketDccComms(DCCComms):
 
-    def __init__(self, url):
+    def __init__(self, url, verify_cert, identity=None):
```

Comment:
If verify_cert is True, identity should not be None. I think we should have this check somewhere.